fishtank/src/cluster.test.ts

```diff
@@ -40,19 +40,110 @@ describe('Cluster', () => {
       const containerDatadir = join(
         tmpdir(),
         'fishtank',
-        'my-test-cluster_my-test-container',
+        'my-test-cluster',
+        'my-test-container',
         '.ironfish',
       )
       const volumes = new Map<string, string>([[containerDatadir, '/root/.ironfish']])
+
       expect(
         await promises.readFile(resolve(containerDatadir, 'config.json'), { encoding: 'utf8' }),
       ).toEqual('{"networkId":0}')
+
+      expect(runDetached).toHaveBeenCalledWith('ironfish:latest', {
+        name: 'my-test-cluster_my-test-container',
+        networks: ['my-test-cluster'],
+        hostname: 'my-test-container',
+        labels: { 'fishtank.cluster': 'my-test-cluster' },
+        volumes: volumes,
+      })
+    })
+
+    it('launches a detached container with custom network definition', async () => {
+      const backend = new Docker()
+      const cluster = new Cluster({ name: 'my-test-cluster', backend })
+
+      const runDetached = jest.spyOn(backend, 'runDetached').mockReturnValue(Promise.resolve())
+
+      const networkDefinition: Partial<NetworkDefinition> = {
+        id: 0,
```

Comment:
Nit: let's specify a non-default value (like 123) so that we can make sure that this input is really being used, and we're not observing some default behavior